iele.md

```diff
@@ -429,13 +384,13 @@ Some checks if an opcode will throw an exception are relatively quick and done u
     syntax InternalOp ::= "#badJumpDest?" "[" Op "]"
  // ----------------------------------------------------
     rule <k> #badJumpDest? [ OP                        ] => . ... </k> requires notBool isJumpOp(#code(OP))
-    rule <k> #badJumpDest? [ JUMP (LABEL)              ] => . ... </k> <jumpTable> JUMPS </jumpTable> requires LABEL in_keys(JUMPS)
-    rule <k> #badJumpDest? [ LOCALCALL (LABEL,_,_) _ _ ] => . ... </k> <jumpTable> JUMPS </jumpTable> requires LABEL in_keys(JUMPS)
-    rule <k> #badJumpDest? [ JUMPI(LABEL) _            ] => . ... </k> <jumpTable> JUMPS </jumpTable> requires LABEL in_keys(JUMPS)
+    rule <k> #badJumpDest? [ JUMP (LABEL)              ] => . ... </k> <fid> FUNC </fid> <funcId> FUNC </funcId> <jumpTable> JUMPS </jumpTable> requires LABEL in_keys(JUMPS)
```

Comment:
This code looks slightly tricky - it's getting `FUNC` from the `<fid>` cell and then relying on context abstraction between the `<funcId>` and `<jumpTable>` cells to make sure it gets correct jump table, right? It would be clearer to write more explicitly like
`<function>... <funcId> FUNC </funcId> <jumpTable> JUMPS </jumpTable> ...</function>`